.gitignore

```diff
@@ -1,2 +1,3 @@
 .data
 .DS_Store
+.idea
```

Comment:
I use webstorm so I was hoping to add this in if that is okay?

Reply:
Yep, :+1: